recipes/gnucobol/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "gnucobol" %}
+{% set version = "3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 3bb48af46ced4779facf41fdc2ee60e4ccb86eaa99d010b36685315df39c2ee2
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - pkg-config
+    - make
+  host:
+    - gmp
+    - json-c
+    - libxml2
+    - ncurses
+    - libdb
```

This needs libintl on non-Linux platforms.
```suggestion
    - libdb
    - libintl-devel  # [not linux]
```

See in the `osx-64` build:
```
2024-05-03T13:07:27.6609120Z checking for GNU gettext in libintl... no
```